tests/x11/yast2_lan_restart_devices.pm

```diff
@@ -79,37 +88,44 @@ sub add_device {
 
 sub select_special_device_tab {
     my $device = shift;
+
```

Comment:
these whitespace changes seem unrelated. I suggest to revert. If you want to change the style I suggest to change the perltidy file or suggest a change valid for all test modules.

Reply:
I modified the function/file so I guess I can apply my own style as there is not rigid convention about this thing AFIK and in the repo there are many example of both styles. For me the point to use the space is that in Perl there is not signature of the functions so it is much more clear separate the parameters from the rest of the code.